starknet_py/net/client_models.py

```diff
@@ -346,6 +346,41 @@ class ContractClass:
     abi: Optional[AbiDictList] = None
 
 
+@dataclass
+class NewEntryPoint:
+    """
+    Dataclass representing contract entry point
+    """
+
+    function_idx: int
+    selector: int
+
+
+@dataclass
+class NewEntryPointsByType:
+    """
+    Dataclass representing contract class entrypoints by entry point type
+    """
+
+    constructor: List[NewEntryPoint]
+    external: List[NewEntryPoint]
+    l1_handler: List[NewEntryPoint]
+
+
+@dataclass
+class NewContractClass:
+    """
+    Dataclass representing Cairo1 contract declared to Starknet
+    """
+
+    contract_class_version: str  # Should this be added? Purpose of it is unclear to me.
```

Comment:
Previously, there was `contract class > program > compiler_version`. Maybe it has some similiar functionality.

Reply:
I think we should include it. I cannot come up with any reasonable argumentation for or against it though.